### ISSUE TITLE
[Request] Make Aqua Affinity a passive upgrade

### ISSUE THREAD

**Issue by Fogapod** [MachineMuse/MachineMusePowersuits#970]
**Is your feature request related to a problem? Please describe.**
Aqua affinity is an active module and thus cannot be combined with silk touch upgrade, for example.

**Describe the solution you'd like**
Make Aqua Affinity a passive upgrade.

**Comment by iLLHunter** [comment]
I agree with this, aqua affinity is an enhancement not a tool. Since it only pertains to when the player is in water, the enhancement should be applied (automatically) then
